py/export_conda_env.py

```diff
@@ -1,12 +1,31 @@
 #!/usr/bin/env python3
 import os
+from subprocess import run
 
-cmd = 'conda env export > environment.yml'
 
-print('Running from repo root:\n')
-print(cmd)
-repo_root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
-os.chdir(repo_root)
+export_env_cmd = "conda env export"
+export_out = run(
+    export_env_cmd.split(),
+    capture_output=True, text=True, shell=False
+)
+# Ensure the diff is as expected
+# first and last key should be name and prefix respectively
+lines = export_out.stdout.splitlines()
+name_key = 'name:'
+prefix_key = 'prefix:'
+assert \
+    len(lines) >= 2 \
+    and lines[0][: len(name_key)] == name_key  \
+    and lines[-1][: len(prefix_key)] == prefix_key \
+    , "Environment file format is not supported."
 
-os.system(cmd)
 
+file_path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+    "environment.yml"
+)
```

Comment:
It appears that this script used to live somewhere else, and this triple-dirname is no longer a valid way to get to the repo root. Can you change this to instead make use of `Repo.root()` in `py/util/repo_util.py`?

(I'm surprised the script wrote to the correct yml file with this triple-dirname.)